superset-frontend/src/explore/components/controls/MetricsControl.jsx

```diff
@@ -337,7 +337,11 @@ export default class MetricsControl extends React.PureComponent {
         <OnPasteSelect
           isMulti={this.props.multi}
           name={`select-${this.props.name}`}
-          placeholder={t('choose a column or aggregate function')}
+          placeholder={
+            this.props.multi
+              ? t('choose one or more column or aggregate function')
```

Comment:
`columns or aggregate functions`?

Reply:
I went back and forth on that too, and didn't even find anything definitive in the grammar handbooks, but I agree that the plural sounds more natural .

Reply:
It doesn't seem like there IS a definitive answer... the rules conflict! 

The [Proximity Rule](https://www.thoughtco.com/proximity-agreement-grammar-1691697) says you should go with whichever is closer to the verb. In this case, that'd be "choose", so you would go with "column or agregate function". 

However, the [Notional Agreement](https://www.merriam-webster.com/words-at-play/notional-agreement-subject-verb-principle-proximity) seems to back us up here that these are cognitively a collection, so the plural treatment makes more sense. 

> "Don't take any notice of teachers and textbooks in such matters. Nor of logic. It is good to say 'More than one passenger was hurt,' although more than one equals at least two and therefore logically the verb ought to be plural were not singular was!"

_–– CS Lewis_

+1 for team plural. And yes, I'm going back to what I _should_ be doing, now.

Reply:
ha, that's great. I can also comment in the code that we went with the Notional Agreement here in case anyone wonders. jk :)